templates/compositions/ec-component-organisation-teaser/organisation-teaser.html.twig

```diff
@@ -0,0 +1,117 @@
+{% spaceless %}
+
+{#
+/**
+ * @file
+ * Theme implementation for ECL Organisation Teaser.
+ *
+ * Available variables:
+ * - "name" (string) (default: ''): name of the organisation.
+ * - "acronym" (string) (default: ''): acronym of the organisation.
+ * - "logo" (ImageValueObject) (default: {}): logo of the organisation.
+ * - "organisation_details" (array) (default: []): list of fields.
+ * - "contact_page_url" (string) (default: ''): email adress or contact page to contact the organisation.
+ * - "contact_page" (array) (default: []): processed organisation contact page url.
+ */
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-pb-m' %}
+{% set _extra_attributes = '' %}
+{% set _name = name|default('') %}
+{% set _acronym = acronym|default('') %}
+{% set _logo = logo|default({}) %}
+{% set _organisation_details = organisation_details|default([]) %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Icon for the "Contact organisation" link. #}
+
+{% if contact_page_url is not empty %}
+  {% set _contact_page_url = contact_page.url|default('') %}
+  {% set _contact_page_icon = contact_page.icon|default('internal') %}
+  {% set _contact_page_label = contact_page.label|default('Contact organisation'|t) %}
+
+  {% if _contact_page_icon == 'internal' %}
+    {% set _icon = {
+      path: ecl_icon_path,
+      type: 'ui',
+      name: 'corner-arrow',
+      size: 's',
+      color: 'primary',
+      transform: 'rotate-90'
+    }
+    %}
+  {% else %}
+    {% set _icon = {
+      path: ecl_icon_path,
+      type: 'ui',
+      name: _contact_page_icon,
+      size: 's',
+      color: 'primary'
+    }
+    %}
+  {% endif %}
+{% endif %}
+
+{# Create macro for an image. #}
+
+{% macro image(src, alt) %}
+  <div role="img"
+       aria-label="{{ alt }}"
+       class="ecl-u-flex-shrink-0
+        ecl-u-mr-s
+        ecl-u-media-a-s
+        ecl-u-media-bg-size-contain
+        ecl-u-media-bg-repeat-none
+        ecl-u-d-lg-block"
+        style="background-image: url('{{ src }}');"
+  ></div>
+{% endmacro %}
```

Comment:
This is used only once, so let's not wrap it in a macro, we can just use it in place where it's needed.